bootstrap/README.md

```diff
@@ -50,9 +56,9 @@ If you have plugged in your speaker via the headphone jack, you may experience i
 
 8. Run a recipe
 
-    cd ~/Desktop/tjbot/recipes/intro
```

Comment:
maybe we should make the `intro` script part of bootstrap?  @maryamashoori didn't want to include it as a mainline recipe, but I feel we should have a script that runs and indicates some kind of success that bootstrap worked.